data_specification/enums/data_type.py

```diff
@@ -45,6 +45,8 @@ class DataType(Enum):
              False,
              int,
              np.uint8,
+             'u',
+             1,
```

Comment:
Can you describe what these two new entries in the data types are in the description at the top of the file?  Otherwise, this looks fine to me.

Reply:
Removed now so no longer necessary